docker/README.md

```diff
@@ -0,0 +1,45 @@
+# PhoXiControl inside the docker image
+This is a short manual on deploying the PhoXiControl inside the a docker image
+
+## Before build
+Select a desired Ubuntu 18.04 installer and copy it to the `context/installer/phoxi.run` file.
+
+
+
+## Build
+
+Once the installer is present, proceed with the image build process:
+
+```bash
+DOCKER_BUILDKIT=1 docker build --squash -t "photoneo/phoxicontrol:latest" -f Dockerfile context
+```
+
+
+
+## Run
+
+In order to be able to run the image, `docker` must be set up to be able to pass `nvidia` runtime.
```

Comment:
I would add a section or a link on how to install Nvidia runtime https://wiki.ra.photoneo.com/en/DevOps/nvidia-runtime-support